Makefile

```diff
@@ -1,71 +1,86 @@
 # System Setup
 SHELL = bash
 
-RUST_VERSION ?= stable
+
+.PHONY: help
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+
+service_crategen/botocore/.git:
+	git submodule update --init --recursive
```

Comment:
Adding this makes the first attempt to generate services seamless. Previously people would usually have to parse error messages stating that botocore is missing.